twat.py

```diff
@@ -315,10 +343,12 @@ def extract_twat(soup, twats, timestamp):
 
 			tweet_id = div.attrs["data-tweet-id"]
 			tweet_user = div.attrs["data-screen-name"]
+
```

Comment:
please remove these superfluous whitespace changes from the commit that introduced them (it is probably due to squashing)